README.md

```diff
@@ -1,93 +1,92 @@
-## Digital Publishing Concourse Tools
+# Digital Publishing Concourse Tools
 
 Build tools we use in our Concourse CI pipelines as docker images.
 
-### Getting started
+All images are currently published to Docker Hub. You will require a login to the [onsdigital](https://hub.docker.com/orgs/onsdigital/repositories) Docker Hub organisation to view as well as write permissions to push changes to account. (The write permissions are usually restricted to TLs and Platform team)
```

Comment:
https://hub.docker.com/orgs/onsdigital/repositories ==> this link is erroring

Reply:
@sandypadmanabhan our repos are private can you log in to docker hub and try again